taskcluster/rb_taskgraph/job.py

```diff
@@ -96,7 +95,14 @@ def _extract_gradlew_command(run):
         _generate_secret_command(secret) for secret in run.get("secrets", [])
     ]
 
-    gradle_command = ["./gradlew"] + run.pop("gradlew")
+    maven_dependencies_dir = path.join(fetches_dir, "android-gradle-dependencies")
+    gradle_repos_args = [
+        "-P{repo_name}Repo=file://{dir}/{repo_name}".format(
+            dir=maven_dependencies_dir, repo_name=repo_name
+        )
+        for repo_name in ("google", "jcenter")
+    ]
+    gradle_command = ["./gradlew"] + gradle_repos_args + ["listRepositories"] + run.pop("gradlew")
```

Comment:
For debugging purpose each gradle job will output the list of repositories it used before running anything.